src/codeu/chat/common/ServerInfo.java

```diff
@@ -26,7 +26,11 @@ public ServerInfo(Uuid version, Time startTime){
 		this.version = version;
 		this.startTime = startTime;
 	}
-    public long upTimeInSec(){
-		return (Time.now().inMs()-(startTime).inMs())/1000;
+    public String upTime(){
+		long seconds = (Time.now().inMs()-(startTime).inMs())/1000;

```

Comment:
While this is correct, I suggest looking at 
https://stackoverflow.com/questions/11357945/java-convert-seconds-into-day-hour-minute-and-seconds-using-timeunit
And using the suggested method there (readability)